README.md

```diff
@@ -93,34 +93,15 @@ klonol -h
 
 ## Usage
 
-### Setting the environment variables
-
-For GitHub the following variables need to be set.
+### Setting the variables
 
+The following variables need to be set in a file called `credentials.toml`.
 | Name                | Description                                                                              | Compulsory |
 | ------------------- | ---------------------------------------------------------------------------------------- | ---------- |
-| GITHUB_USERNAME     | The username whose repositories are to be queried                                        | Yes        |
-| GITHUB_ACCESS_TOKEN | The personal access token generated previously                                           | Yes        |
-| GITHUB_BASE_URL     | The base domain to be used to test SSH and make API calls from. Defaults to `github.com` | No         |
-
-For Gitea the following need to be set:
-
-| Name               | Description                                                                                  | Compulsory |
-| ------------------ | -------------------------------------------------------------------------------------------- | ---------- |
-| GITEA_USERNAME     | The username whose repositories are to be queried                                            | Yes        |
-| GITEA_ACCESS_TOKEN | The personal access token generated previously                                               | Yes        |
-| GITEA_BASE_URL     | The domain where the Gitea instance is hosted. Do not include the protocol (e.g. `https://`) | Yes        |
```

Comment:
This is vital information. Should not be deleted. People self-hosting Gitea will need to provide this information.